tests/perf/mt_ubench_l3.cpp

```diff
@@ -1,35 +1,59 @@
-#include <assert.h>
-#include <pthread.h>
-#include <stdlib.h>
+/**
+ * *****************************************************************************
+ * \file perf/mt_ubench_l3.cpp
+ * \author Greg Law
+ * \brief L3: Fast-logging micro perf-benchmarking
+ * \version 0.1
+ * \date 2024-06-25
+ *
+ * \copyright Copyright (c) 2024
+ *
+ * Usage: program-name [ <number-of-threads> ]
+ * *****************************************************************************
+ */
 #include <stdio.h>
-#include <sys/syscall.h>
+#include <stdlib.h>
 #include <unistd.h>
+#include <assert.h>
+#include <sys/syscall.h>
+#include <pthread.h>
 
 #include "l3.h"
 
-int gettid() {
-    return syscall(SYS_gettid);
-}
```

Comment:
Deleted because `gettid()` is not used in code base. And this causes Mac/OSX builds to fail on CI.